CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - #974, Fix RPC error when function has single OUT param - @steve-chavez
 - #1021, Reduce join size in allColumns for faster program start - @nextstopsun
 - #411, Remove the need for pk in &select for parent embed - @steve-chavez
+- #925, Fix RPC high memory usage by using parametrized query - @steve-chavez
```

Comment:
Is it also worth noting how you changed the behavior with overloaded procs?

Reply:
In which way?(I'm assuming changelog is related to functionality for end users)

The detection of the `return type/volatility` is working the same as before(HashMap used to merge duplicates `ProcDescription` into just one) and the `callProc` function considers overloaded functions(that previously were undocumented/untested).

Reply:
Nevermind, if it caused no visible change then nothing to document. I wasn't sure how it might affect using overloaded functions.